web/src/utils/functions.ts

```diff
@@ -33,16 +53,33 @@ function getFunctionType(data: FunctionData): FunctionType {
 }
 function getFunctionTypePretty(data: FunctionData): string {
     const funcType = getFunctionType(data);
-    return functionTypePrettyName[funcType] ?? 'Server-side';
+    return functionTypePrettyName[funcType];
 }
 
-export function getFunctionInfo(data: FunctionData): any {
+export type FunctionInfo = {
+    description: string;
+    type: FunctionType;
+    typePretty: string;
+    pair: boolean;
+    examples: { code: string; description?: string }[];
+    notes?: string[];
+    important_notes?: string[];
```

Comment:
As I said, important_notes seems unnecessary. Just improve the notes attribute in the function schema and the function yamls to include a note type (that decides its color maybe).

Reply:
That's a great observation, I'll update the structure to make it acceptable to specify a type for the note, changing only a few visual details.